Stripe/PKPaymentAuthorizationViewController+Stripe_Blocks.m

```diff
@@ -39,28 +38,23 @@ - (void)paymentAuthorizationViewController:(__unused PKPaymentAuthorizationViewC
                        didAuthorizePayment:(PKPayment *)payment completion:(STPPaymentAuthorizationStatusCallback)completion {
     self.onPaymentAuthorization(payment);
 
-    void(^tokenOrSourceCompletion)(id<STPSourceProtocol>, NSError *) = ^(id<STPSourceProtocol> result, NSError *error) {
-        if (error) {
-            self.lastError = error;
+    void(^paymentMethodCreateCompletion)(STPPaymentMethod *, NSError *) = ^(STPPaymentMethod *result, NSError *paymentMethodCreateError) {
+        if (paymentMethodCreateError) {
+            self.lastError = paymentMethodCreateError;
             completion(PKPaymentAuthorizationStatusFailure);
             return;
         }
-        self.onSourceCreation(result, ^(NSError *sourceCreation){
-            if (sourceCreation) {
-                self.lastError = sourceCreation;
+        self.onPaymentMethodCreation(result, ^(NSError *error) {
+            if (error) {
+                self.lastError = error;
                 completion(PKPaymentAuthorizationStatusFailure);
                 return;
             }
             self.didSucceed = YES;
             completion(PKPaymentAuthorizationStatusSuccess);
         });
     };
-    if (self.createSource) {
-        [self.apiClient createSourceWithPayment:payment completion:(STPSourceCompletionBlock)tokenOrSourceCompletion];
-    }
-    else {
-        [self.apiClient createTokenWithPayment:payment completion:(STPTokenCompletionBlock)tokenOrSourceCompletion];
```

Comment:
wondering about the implications of removing token creation...Do you know what/if any use cases this was supporting?

Reply:
It looks like it was first written using tokens, then we added configurable support for Source https://github.com/stripe/stripe-ios/pull/892.  I'd guess we didn't remove tokens then to make this a not-breaking change.